src/dist/replication/lib/replica_config.cpp

```diff
@@ -399,18 +399,20 @@ void replica::on_update_configuration_on_meta_server_reply(error_code err, dsn_m
         if (err != ERR_INVALID_VERSION)
         {
             rpc_address target(_stub->_failure_detector->get_servers());
-            _primary_states.reconfiguration_task = rpc::call(
-                target,
+            _primary_states.reconfiguration_task = rpc::create_rpc_response_task(
                 request,
                 this,
                 [this, req](error_code err, dsn_message_t request, dsn_message_t response)
                 {
                     on_update_configuration_on_meta_server_reply(err, request, response, std::move(req));
                 },
                 gpid_to_hash(get_gpid())
-                );
+            );
```

Comment:
@shengofsun you can add additional timeout parameter in rpc::call instead of introducing new system call as dsn_task_set_delay.